app/mixins/sortable-by-position.js

```diff
@@ -0,0 +1,35 @@
+import Ember from 'ember';
+
+const { Mixin } = Ember;
+
+export default Mixin.create({
+
+  /**
+   * Callback function for <code>Array.sort()<code>.
+   *
+   * @method learningMaterialSortingCallback
+   * @param {Ember.Object} obj1
+   * @param {Ember.Object} obj2
+   * @return {Number}
+   */
+  learningMaterialSortingCallback(obj1, obj2) {
```

Comment:
Given that this is generic and probably re-usable should it just be called positionSortingCallback?